music21/harmony.py

```diff
@@ -1804,8 +1808,8 @@ def _parseFigure(self):
                 # remove the root and bass from the string and any additions/omissions/alterations/
                 st = prelimFigure.replace(m1.group(), '')
             else:
-                raise ValueError  # This means that the given argument wasn't
-                # a proper chord name.
+                raise ValueError(f"Chord {prelimFigure} does not begin "
+                    f"with a valid root note")
```

Comment:
single quotes; no f-string on second , and ensure that strings aline on consecutive lines.  e,g, : 

```
                raise ValueError(
                      f'Chord {prelimFigure} does not begin '
                      + 'with a valid root note'
                )
```